docs/batch.md

```diff
@@ -0,0 +1,86 @@
+# PREVIEW - Batch Messages (A.K.A. Fast Sync)
```

Comment:
```suggestion
# Batch Messages
```

As a page title, I think it's best to keep this simple. (I appreciate though that the preview warning is top of page just below the title.)

Also, since "Fast Sync" is technically the Pipelinewise spec name, we should avoid the name in technical documentation since we are not technically compatible with that separate spec - even though we are inspired by it.